components/GatekeeperConfig.vue

```diff
@@ -408,11 +412,8 @@ export default {
           </button>
           <AsyncButton
             :mode="mode"
```

Comment:
`mode` is perhaps a confusing word here given what it means elsewhere, but all it means to an AsyncButton is which set of words to use.  So there's no need for a new `use-enable-label` or checking if it's `create` inside the component, you can just pass `mode="enable"` here.

Reply:
2) Then set `mode="enable"` here.  It's an arbitrary key in the `LABELS` map, it _doesn't have to_ correspond to one of the modes that are used by `create-edit-view`.